src/main/java/org/kainos/ea/cli/Band.java

```diff
@@ -0,0 +1,37 @@
+package org.kainos.ea.cli;
+
+public class Band {
+    private String BandName;
+    private String BandLevel;
+    private String Competencies;
```

Comment:
Thought we agreed these should start with lowercase? Very unusual! Makes these look like class references!